Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3
+ADD generate.py /
+ENV TOKEN $TOKEN
+ENV BUCKET $BUCKET
+ENV GOOGLE_APPLICATION_CREDENTIALS $GOOGLE_APPLICATION_CREDENTIALS
+RUN pip install --upgrade pip
+RUN pip install --no-cache-dir git+git://github.com/puppetlabs/infinitory.git@master
+ENTRYPOINT python generate.py pe-infranext-prod.infc-aws.puppet.net ${TOKEN} ${BUCKET}
```

Comment:
Should the fqdn be hard coded here or passed in like token is? I feel kinda like it should be passed in.

Reply:
yep, you're absolutely correct